site/docs/v1/tech/lambdas/ldap-connector-reconcile.adoc

```diff
@@ -32,22 +32,22 @@ The FusionAuth user object is well documented the link:../apis/users[User API] d
 
 LDAP attributes can be returned to FusionAuth in a string form or a byte array. Some attributes are considered non-string values and need to be provided in a byte array to be useful in the Lambda function.
 
-A non-string attribute should be requested as a byte array. To request an attribute as a byte array, use the `;binary` LDAP attribute option as a suffix on your requested attribute. For example, instead of requesting `objectGuid`, you will request `objectGuid;binary`.
+A non-string attribute should be requested as a byte array. To request an attribute as a byte array, use the `;binary` LDAP attribute option as a suffix on your requested attribute. For example, instead of requesting `objectGUID`, you will request `objectGUID;binary`.
```

Comment:
This is fine. I think I had just used the same variable name I saw from AD. Is `objectGUID` a more common variable name than `objectGuid`?

Reply:
I don't have a lot of data points, but both Bradley Kite and my Active Directory servers used `objectGUID` and that is what is specified by the MS docs: https://docs.microsoft.com/en-us/windows/win32/adschema/a-objectguid